Signal/src/environment/NotificationsManager.m

```diff
@@ -155,6 +154,11 @@ - (void)notifyUserForIncomingMessage:(TSIncomingMessage *)message
     }
 
     NSString *messageDescription = message.description;
+    NSString *senderName = [contactsManager displayNameForPhoneIdentifier:message.authorId];
+    NSString *groupName = [thread.name stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceCharacterSet]];
```

Comment:
> This name is a little confusing, because it will be populated with the contact's name for ContactThreads, right?

True, but we only use the groupName when displaying notifications for groups.